frameworks/kafka/tests/test_upgrade.py

```diff
@@ -13,11 +13,11 @@
 
 @pytest.mark.dcos_min_version("1.10")
 @sdk_utils.dcos_ee_only
-def test_upgrade_110_210(configure_security):
+def test_upgrade_111_210(configure_security):
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
     sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
 
-    from_version = '2.3.0-1.1.0'
+    from_version = '2.4.0-1.1.1'
     to_version = 'stub-universe'
```

Comment:
@shubhanilBag this will fail again when we do next release of the Kafka

should we calculate the second last version? or maybe pin it for the test?

Reply:
It will only update to the lowest *-2.1.0 version

Reply:
perfect! didn't see it